extensions/azurecore/src/azureResource/providers/postgresArcServer/postgresServerTreeDataProvider.ts

```diff
@@ -45,7 +45,7 @@ export class PostgresServerArcTreeDataProvider extends ResourceTreeDataProviderB
 				databaseName: databaseServer.defaultDatabaseName,
 				userName: `${databaseServer.loginName}@${databaseServer.fullName}`,
 				password: '',
-				authenticationType: 'SqlLogin',
+				authenticationType: 'AzureMFA',
```

Comment:
Did you check that this actually works? The PGSQL team just recently reverted an update to their extension to add AAD support because of a bug so not sure if there's any issues here

Reply:
When I was writing this it did nothing (didn't change the behavior at all) but with the new update this should integrate with it!